scripts/js/lib/api/processHtml.ts

```diff
@@ -346,7 +347,11 @@ export async function processMembersAndSetMeta(
   $: CheerioAPI,
   $main: Cheerio<any>,
   meta: Metadata,
-  options: { isCApi: boolean },
+  options: {
+    isCApi: boolean;
+    pkgName: string;
+    kebabCaseAndShorten: boolean;
+  },
```

Comment:
This type gets used a lot. It might be worth DRYing. I'm not sure what a good name would be, maybe `ProcessHtmlOptions` or something related to the main function all these args get sent to?